src/editor/style-cards/maxiStyleCardsEditor.js

```
@@ -265,24 +395,132 @@ const MaxiStyleCardsEditor = forwardRef(({ styleCards, setIsVisible }, ref) => {
+			const customColorItems = styleCardsTabNode.querySelectorAll(
+				'.maxi-style-cards__custom-color-presets__box'
+			);
+			if (customColorItems && customColorItems.length > 0) {
+				uiCustomColors = Array.from(customColorItems).map(item => {
+					const colorSpan = item.querySelector(
+						'.maxi-style-cards__custom-color-presets__box__item'
+					);
+					const colorName = item.getAttribute('title') || '';
+					const colorIdAttr = item.getAttribute('key'); // e.g., maxi-style-cards__custom-color-presets__box-1234567890
+					let colorId = null;
+					if (colorIdAttr && colorIdAttr.includes('-')) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)